src/Stringy.php

```diff
@@ -1205,6 +1206,7 @@ public function removeLeft($substring)
      */
     public function removeRight($substring)
     {
+        $substring = strval($substring);
```

Comment:
Yea, I was going to comment on the use of `strval`, but stopped since an updated fix wouldn't touch that line at all. A second reason for casting instead of the function call, is that it fits with the rest of the code.